Rogue-Ape-Studios-XR/Secrets-of-Ignacios
# ✨grimiore summoning

## Content

Added the ability for the player to summon the grimiore and made appropriate prefabs for further development

## Changes

### Added
`Prefabs/Grimiore` : Was created. A fully functional prefab that has the book and bracelet holder so that the player can 
`Prefabs/BookHolder` : Was created. A prefab that holds the grimiore, which will later be used for the player to store learned spells and look back when in need.
`Prefabs/Bracelet Holder` : Was created. A prefab that holds a button which can appear on the back of the player's hand.
`Prefabs/HeadGazeInteractor` : Was created. This object is placed on the player to track if the player is looking at certain objects
`Scripts/Bracelet Holder` : Was created. You can add the purpose here.

### Updated
`MR Interaction setup` : Was updated with the grimiore and headgazeinteractor. Also there are transforms added to the poke interactors so that the index fingers can interact with UI elements

## Testing

The feature / Bugfix can be testing by following these steps:

1. Enter a scene with the updated MR interaction setup
2. Run it in vr
3. Look at your the back of your left hand, a button should appear
4. Press the button, a book should appear, and pressing it again should be make it disappear again
